middlewares/validators/user.js

```diff
@@ -77,8 +77,51 @@ const validateJoinData = async (req, res, next) => {
     res.boom.badRequest(error.details[0].message);
   }
 };
+
+/**
+ * Validates getting users query
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ * @param next {Object} - Express middelware function
+ */
+async function getUsers(req, res, next) {
+  const schema = joi
+    .object()
+    .strict()
+    .keys({
+      size: joi
+        .string()
+        .optional()
+        .pattern(/^[1-9]\d?$|^100$/)
+        .messages({
+          "string.base": "size should be a type of string",
+          "string.empty": "size must contain value in range 1-100",
+          "string.pattern.base": "size must be in range 1-100",
+          "any.required": "size is required",
```

Comment:
I don't think we should make it mandatory for the user to pass `size` always. by default, the size is 100 if not provided by the user.

Reply:
If the size is not provided, then there should be a Default/ideal size; here, maybe we can say 10 records; 
we should not put max size as the default or ideal size. It will increase the unnecessary load. 
If the user wants more records, we respond to requests.

Reply:
`size` is optional and default value is 10.

Reply:
I checked this point with Ankush today. Changing the default size would unnecessarily lead to more network calls. Hence we are going to stick with 100 as earlier

Reply:
Changed backed to 100.